src/main/java/com/thoughtworks/pacman/core/Ghosts.java

```diff
@@ -51,11 +51,11 @@ public void freeGhostsBasedOnScore(int score) {
         }        
     }
 
-    public void advance(long timeDeltaInMillis) {
-        blinky.advance(timeDeltaInMillis);
-        pinky.advance(timeDeltaInMillis);
-        inky.advance(timeDeltaInMillis);
-        clyde.advance(timeDeltaInMillis);
+    public void advance(long timeDeltaInMillis,boolean isItStopped) {
+        blinky.advance(timeDeltaInMillis,isItStopped);
```

Comment:
missing space in this line and the following lines

`,isItStopped`

=>

`, isItStopped`